doc/tutorial/index.rst

```diff
@@ -18,6 +21,20 @@ project and can be built and played with locally as follows:
    # load in ghci to play with it
    $ cabal new-repl tutorial
 
+Stack
+--------
+
+To build the tutorial using `stack <https://docs.haskellstack.org/en/stable/README/>`_ you can run the following:
+
+.. code-block:: bash
```

Comment:
The instructions below seem to be about setting up a new Stack project for a servant application, while the instructions for `cabal-install` are specifically about _building the tutorial_...?

Reply:
Hey, thanks for the response - happy to trim this one up today. I was just following a chat [here](https://github.com/haskell-servant/servant/issues/1101). It would create a new project using the servant template, which includes the tutorial endpoint. From personal experience, this was the quickest way to get it running with stack.

Would you prefer if I try an alternative route for getting it up and running? Alternatively, I could make the wording a bit more explicit:

```
The servant stack template includes the working tutorial. To initialize this template, run:
```

Happy to do anything you'd like - first-time contributing so apologies if I've dropped the ball on this.

Reply:
Yes, I'm happy to keep your original idea but with the revised wording, thanks!

Reply:
Awesome - thanks! Have updated. LMK if there's anything else I can do to help.